ui/src/App.tsx

```diff
@@ -233,41 +238,49 @@ export function App() {
     ];
 
     const handleNavigate = (row) => async () => {
+        track({ action: "ViewVolumeDetails" });
         ddClient.desktopUI.navigate.viewVolume(row.volumeName);
     };
 
     const handleClone = (row) => () => {
+        track({ action: "CloneVolumePopup" });
```

Comment:
Should we remove the `Popup` suffix of all the actions? I think it'd be best not to tie the action to any element in the UI in case we don't use popups in the future.

Reply:
I wanted to distinguish between opening the popup and finally running the action (when the user clicks OK), or cancel